### ISSUE TITLE
Query param bug in routing example app

### ISSUE THREAD

**Issue by patsy-issa** [simplabs/ember-workshop#364]
Steps to reproduce:

- Checkout repo
- Navigate to the routing example folder and launch the application
- Navigate to `/media`
- click on the movies link
- select an item from the movie list

During the transition the list will revert back to the albums for a short duration before transitioning into the correct route.

**Comment by marcoow** [comment]
@patsy-issa didn't you say this was caused by a bug in Ember itself?

**Comment by patsy-issa** [comment]
That is correct, it seems similar to https://github.com/emberjs/ember.js/issues/12107 I have extracted the repro example and writing up my findings to submit them to the ember.js repo

**Comment by marcoow** [comment]
Ok, please link here then so we can keep track.

**Comment by patsy-issa** [comment]
Ember repo issue: https://github.com/emberjs/ember.js/issues/19307

**Comment by sly7-7** [comment]
If that could help, I've found a workaround (posted in the related issue)
